perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -0,0 +1,150 @@
+#/!/bin/bash
+################################################
+## Auth=mifiedle@redhat.com lhorsley@redhat.com
+## Desription: On a 10 node OVN cluster, time how long it takes to scale up to 2000 pods with and without the networkpolicy 
+## Note: The scale up time with the networkpolicy in place will be more than without, but it should not be an order of magnitude difference.	
+## This automated test does not cover step 5.1 - 5.4 in the Polarian test case
+## Polarion test case: OCP-41535 - NetworkPolicy scalability - 2000 pods per namespace using customer network policy	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-41535
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/node-affinity-anti-affinity-config.yml
+################################################ 
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source network_policy_scalability_env.sh
+
+pod_scale_max=2000
+pod_scale_min=500
+pass_fail=0
+project_name="np-issue-test"
+deployment_name=$project_name
+export NAME=$project_name
+export NAMESPACE=$project_name
+
+
+project_label="test=np-issue-test"
+
+
+
+echo "Create and label new project"
+prepare_project ${NAMESPACE} $project_label
```

Comment:
In this function you'll need to update to have the proper privileges to get ride of this error message: 

```
01-31 13:39:44.745  time="2023-01-31 18:39:44" level=info msg="Waiting up to 3h0m0s all job actions to be completed"
01-31 13:39:44.745  W0131 18:39:44.538800    2218 warnings.go:70] would violate PodSecurity "restricted:latest": seccompProfile (pod or container "pause" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
01-31 13:39:44.745  time="2023-01-31 18:39:44" level=info msg="Actions in namespace np-issue-test completed"
```

You'll probably want a namespace yaml file template like the below
```
apiVersion: v1
kind: Namespace
metadata:
  labels:
    kubernetes.io/metadata.name: ${NAMESPACE}
    pod-security.kubernetes.io/audit: privileged
    pod-security.kubernetes.io/enforce: privileged
    pod-security.kubernetes.io/enforce-version: v1.24
    pod-security.kubernetes.io/warn: privileged
    security.openshift.io/scc.podSecurityLabelSync: “false”
  name: ${NAMESPACE}
```

Reply:
Sorry looking more I don't think you want to create the project yourself before you run, the run_workload will create it properly for you

`"Namespace np-issue-test already exists"`